### PR TITLE
ascanrulesBeta: Tweak for SourceCodeDisclosureFileInclusionScanRule alert details

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Add maint note.
- Message.properties > Removed unused resource keys/values.
- SourceCodeDisclosureFileInclusionScanRule > Tweaked so that the example alerts use the same details as the plugin itself (which were based on vulnerabilities.xml content).

## Related Issues
https://github.com/zaproxy/zaproxy-website/pull/2007

## Checklist
- [ ] Update help - N/A
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [ ] Check code coverage - N/A
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties [hunk 1]
```diff
@@ -218,15 +218,12 @@ ascanbeta.shellshock.ref = http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2
 ascanbeta.shellshock.soln = Update Bash on the server to the latest version
 ascanbeta.shellshock.timingbased.evidence = Using the attack, a delay of [{0}] milliseconds was induced and detected
 
-ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server
+ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server.
 ascanbeta.sourcecodedisclosure.gitbased.evidence = The source code for [{0}] was extracted using [{1}]
-ascanbeta.sourcecodedisclosure.gitbased.extrainfo =
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties:223]: This was unused and unnecessary.

File: addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties [hunk 2]
```diff
@@ -218,15 +218,12 @@ ascanbeta.shellshock.ref = http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2
 ascanbeta.shellshock.soln = Update Bash on the server to the latest version
 ascanbeta.shellshock.timingbased.evidence = Using the attack, a delay of [{0}] milliseconds was induced and detected
 
-ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server
+ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server.
 ascanbeta.sourcecodedisclosure.gitbased.evidence = The source code for [{0}] was extracted using [{1}]
-ascanbeta.sourcecodedisclosure.gitbased.extrainfo = 
 ascanbeta.sourcecodedisclosure.gitbased.name = Source Code Disclosure - Git 
 ascanbeta.sourcecodedisclosure.gitbased.soln = Ensure that Git metadata files are not deployed to the web server or application server
-ascanbeta.sourcecodedisclosure.lfibased.evidence =
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties:226]: Setting this blank was pointless.

File: addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties [hunk 3]
```diff
@@ -218,15 +218,12 @@ ascanbeta.shellshock.ref = http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2
...
-ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server
+ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server.
 ascanbeta.sourcecodedisclosure.gitbased.evidence = The source code for [{0}] was extracted using [{1}]
-ascanbeta.sourcecodedisclosure.gitbased.extrainfo = 
 ascanbeta.sourcecodedisclosure.gitbased.name = Source Code Disclosure - Git 
 ascanbeta.sourcecodedisclosure.gitbased.soln = Ensure that Git metadata files are not deployed to the web server or application server
-ascanbeta.sourcecodedisclosure.lfibased.evidence = 
 ascanbeta.sourcecodedisclosure.lfibased.extrainfo = The output for the source code filename [{0}] differs sufficiently from that of the random parameter [{1}], at [{2}%], compared to a threshold of [{3}%]
 ascanbeta.sourcecodedisclosure.lfibased.name = Source Code Disclosure - File Inclusion
-ascanbeta.sourcecodedisclosure.lfibased.soln = Ensure that arbitrary files specified by the user are not included in the output
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties:229]: No longer used.

File: addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties [hunk 4]
```diff
@@ -218,15 +218,12 @@ ascanbeta.shellshock.ref = http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2
 ascanbeta.shellshock.soln = Update Bash on the server to the latest version
 ascanbeta.shellshock.timingbased.evidence = Using the attack, a delay of [{0}] milliseconds was induced and detected
 
-ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server
+ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server.
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties:221]: Added a period to make it into a sentence.

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureFileInclusionScanRule.java [hunk 5]
```diff
@@ -536,11 +536,7 @@ private AlertBuilder createAlert(
                                 NON_EXISTANT_FILENAME,
                                 randomversussourcefilenamematchpercentage,
                                 this.thresholdPercentage))
-                .setSolution(
-                        Constant.messages.getString("ascanbeta.sourcecodedisclosure.lfibased.soln"))
-                .setEvidence(
-                        Constant.messages.getString(
-                                "ascanbeta.sourcecodedisclosure.lfibased.evidence"))
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureFileInclusionScanRule.java:543]: This was just uselessly setting blank.

File: addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties (hunk 2 above)
Comment (thc202) [reply addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties:226]: It was pointless defining it, not being blank.

**Comment by thc202** [comment]
Worth updating the tests (either in the example alerts or the actual alerts raised have the expected content).

File: addOns/ascanrulesBeta/CHANGELOG.md [hunk 6]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Maintenance changes.
```
Comment (thc202) [inline addOns/ascanrulesBeta/CHANGELOG.md:8]: I'd be more clear this will affect what the user sees.

Comment (kingthorin) [reply addOns/ascanrulesBeta/CHANGELOG.md]: Sure

**Comment by kingthorin** [comment]
Done

**Review by thc202 — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

**Comment by psiinon** [comment]
Looks good but now conflicting

**Comment by kingthorin** [comment]
On it.

**Comment by kingthorin** [comment]
Fixed

**Pull request merged**